contracts/periphery/Periphery.sol

```diff
@@ -176,10 +176,16 @@ contract Periphery is IPeriphery {
 
         // if WETH pools, accept deposit only in ETH
         if (address(underlyingToken) == address(_weth)) {
-            require(marginDelta <= 0, "INV");
+            require(
```

Comment:
Just a general best practice, make sure that the signals are green before sending PR.

Reply:
I picked up your comments and actioned them. One question, when you push to this repo it runs the linter again but those formatting changes are not included in the push. Shall I push again with the linter changes? What is the common practice regarding this?